.github/workflows/ci-pytorch-dockers.yml

```diff
@@ -34,7 +34,6 @@ jobs:
         include:
           # We only release one docker image per PyTorch version.
           # The matrix here is the same as the one in release-docker.yml.
-          - {python_version: "3.9", pytorch_version: "1.9", cuda_version: "11.1.1"}
```

Comment:
Are we fine with dropping pre-built dockers for a currently supported combination of versions?

Reply:
yes, because the only drawback is is that some dependencies are missing so they will be installed in CI, but that is fine...